ctrl:
  ros__parameters:
    freq: 30
    angle_range: [-60, 60]
    pwm_range: [21500, 51500]
    
    start_wing: [1.0, 30.0, 0.0, 0.0, 0.0 ] # | omega_m | A_m | A_d | alpha_m | alpha_d |
    start_flap: [30.0, 0.0]           # | theta_m | theta_d |

    clamp_flap: [-40, 40]

